miqa/core/migrations/0025_globalsettings.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 3.2.11 on 2022-01-21 14:23
+
+from django.db import migrations, models
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+
+def migrate_global_project(apps: StateApps, schema_editor: BaseDatabaseSchemaEditor):
+    Project = apps.get_model('core', 'Project')  # noqa: N806
+    GlobalSettings = apps.get_model('core', 'GlobalSettings')  # noqa: N806
+
+    if global_project := Project.objects.filter(name='global').first():
+        gs = GlobalSettings.load()
+        gs.import_path = global_project.import_path
+        gs.export_path = global_project.export_path
+        gs.save()
+        global_project.delete()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0024_alter_frame_raw_path'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='GlobalSettings',
+            fields=[
+                ('id', models.PositiveSmallIntegerField(primary_key=True, serialize=False)),
+                ('import_path', models.CharField(blank=True, default='', max_length=1000)),
+                ('export_path', models.CharField(blank=True, default='', max_length=1000)),
+            ],
+        ),
+        migrations.RunPython(migrate_global_project),
+    ]
```

Comment:
I think you also need to remove `global_import_export` from the `Project` model

Reply:
Good point. And with the new migration, I figured we could use a squash as well. changed in [b05b85c](https://github.com/OpenImaging/miqa/pull/292/commits/b05b85ca53839899deb48d13b37bca03d683b782)